src/R/Support/Impl/Help/Packages/PackageInfo.cs

```diff
@@ -60,9 +59,7 @@ internal sealed class PackageInfo : NamedItemInfo, IPackageInfo {
                         }
                     }
                     _saved = true;
-                } catch (Exception ex) when (ex is IOException || ex is UnauthorizedAccessException) {
-                    EditorShell.Current.Services.Log.Write(LogVerbosity.Normal, MessageCategory.Error, ex.Message);
-                }
+                } catch (Exception ex) when (ex is IOException || ex is UnauthorizedAccessException) { }
```

Comment:
It is hard to plumb it here at the moment. When I get services attached to buffer/view I will get it back